app/init.jsx

```diff
@@ -16,7 +16,20 @@ Sentry.init({ dsn: `https://${config.SENTRY_PUBLIC_KEY}@sentry.io/${config.SENTR
 
 const store = configureStore();
 const history = syncHistoryWithStore(browserHistory, store);
-const googleAnalyticsId = (NODE_ENV === 'production' || window.location.host === 'www.askdarcel.org') ? 'UA-116318550-1' : 'UA-116318550-2';
+let googleAnalyticsId;
+
+switch (window.location.host) {
+case 'www.askdarcel.org':
+case 'askdarcel.org':
+  googleAnalyticsId = 'UA-116318550-1';
+  break;
+case 'www.sfserviceguide.org':
+case 'sfserviceguide.org':
```

Comment:
Shouldn't this respect the `MOHCD_DOMAIN`? I pulled out a helper method in https://github.com/ShelterTechSF/askdarcel-web/pull/668 for checking whether you're on the `sfserviceguide` URL, although I'm not really sure what your third case is here.

Reply:
Ah did not know you had added this. For sure, will change this weekend

Currently the third case lumps in staging and local development: TBH pretty useless. There might be some value in staging analytics, but could specifically not set up analytics for anything outside those 3. Figured originally there was no harm in tracking